image/image_test.go

```diff
@@ -54,6 +55,28 @@ func TestMarshalKeyOrder(t *testing.T) {
 	}
 }
 
+func TestImageWithAllMethods(t *testing.T) {
+	var cid = "50a16564e727"
+	var raw = []byte("279533a6d3a9")
```

Comment:
perhaps this should be a more "JSON" value; also; this is already tested in `TestNewFromJSON()` above, so I don't think we need to test that again here